backend/Squirrel.Core/Squirrel.Core.BLL/Services/ProjectService.cs

```diff
@@ -0,0 +1,86 @@
+﻿using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using Squirrel.Core.Common.DTO.Projects;
+using Squirrel.Core.BLL.Interfaces;
+using Squirrel.Core.BLL.Services.Abstract;
+using Squirrel.Core.Common.Enums;
+using Squirrel.Core.DAL.Context;
+using Squirrel.Core.DAL.Entities;
+
+namespace Squirrel.Core.BLL.Services
+{
+    public class ProjectService : BaseService, IProjectService
+    {
+        private readonly SquirrelCoreContext _context;
+        private readonly IMapper _mapper;
+
+        public ProjectService(SquirrelCoreContext context, IMapper mapper): base(context, mapper)
+        {
+            _context = context;
+            _mapper = mapper;
+        }
```

Comment:
You have already inherited BaseService, so you only need to write:
`public SampleService(SquirrelCoreContext context, IMapper mapper) : base(context, mapper) { }`